src/frame/content.rs

```diff
@@ -36,6 +36,14 @@ impl Content {
         }
     }
 
+    /// Returns split values of the `Text` frame or None if the value is not `Text`.
+    pub fn text_values(&self) -> Option<Vec<&str>> {
+        match *self {
+            Content::Text(ref content) => Some(content.split('\0').collect()),
```

Comment:
We can simplify this function by using `Content::text().map()`

Reply:
Thanks, changed the method.